### PR TITLE
Replace checkboxes and radiobuttons with new components from the design system

### PR DESCRIPTION
**Pull request by Magnusrm**
## Description

Implement `RadioGroup` for ControlledRadiogroup component, `CheckboxGroup` for CheckboxesContainerComponent and `RadioButtons` for LikertComponent.

## Related Issue(s)

- closes #{issue number}

## Verification/QA

- Manual functionality testing
  - [x] I have tested these changes manually
  - [ ] Creator of the original issue (or service owner) has been contacted for manual testing (or will be contacted when released in alpha)
  - [ ] No testing done/necessary
- Automated tests
  - [x] Unit test(s) have been added/updated
  - [x] Cypress E2E test(s) have been added/updated
  - [ ] No automatic tests are needed here (no functional changes/additions)
  - [ ] I want someone to help me make some tests
- UU/WCAG ([follow these guidelines](https://aksel.nav.no/god-praksis/artikler/utvikling?tema=universell-utforming) until we have our own)
  - [x] I have tested with a screen reader/keyboard navigation/automated wcag validator
  - [ ] No testing done/necessary (no DOM/visual changes)
  - [ ] I want someone to help me perform accessibility testing
- User documentation @ [altinn-studio-docs](https://github.com/Altinn/altinn-studio-docs)
  - [ ] Has been added/updated
  
  - [x] No functionality has been changed/added, so no documentation is needed
  - [ ] I will do that later/have created an issue
  
- Changes/additions to component properties
  - [ ] Changes are reflected in both `src/layout/layout.d.ts` and `layout.schema.v1.json`, and these are all backwards-compatible
  - [x] No changes made
- Support in Altinn Studio
  - [ ] Issue(s) created for support in Studio
  
  - [x] This change/feature does not require any changes to Altinn Studio
- Sprint board
  - [x] The original issue (or this PR itself) has been added to the Team Apps project and to the current sprint board
  - [ ] I don't have permissions to do that, please help me out
- Labels
  - [x] I have added a `kind/*` label to this PR for proper release notes grouping
  - [ ] I don't have permissions to add labels, please help me out

### CODE REVIEW

**Comment by bjosttveit** [comment]
As a Firefox user, I feel like I always have to ruin everything, but the keyboard focus behaves very poorly in Firefox as of now. Below is a comparison of chrome vs firefox for keyboard navigation.

**Chrome**:

https://user-images.githubusercontent.com/47412359/225621567-f5b8ab19-4f56-4f64-b00f-cfcc4f5ccc5d.mov

----
**Firefox**:

https://user-images.githubusercontent.com/47412359/225621552-fbb25a98-7aa8-40d4-8a96-c9b490670641.mov

**Comment by bjosttveit** [comment]
To follow up on my comment. Turns out that Firefox's experimental `has`-selector still needs a lot of work. Turning off the feature flag `layout.css.has-selector.enabled` (disabled by default) fixed the issue.

File: src/features/form/components/Legend.tsx [hunk 1]
```diff
@@ -47,12 +47,7 @@ export function Legend(props: IFormLegendProps) {
   return (
     <>
       <div className={classes.legendHelpTextContainer}>
-        <label
-          className='a-form-label title-label'
-          htmlFor={props.id}
-        >
-          {LabelText}
-        </label>
+        <legend className='a-form-label title-label'>{LabelText}</legend>
```
Comment (bjosttveit) [inline src/features/form/components/Legend.tsx:50]: Would be nice to get rid of global css whenever we touch code where it is present

Comment (Magnusrm) [reply src/features/form/components/Legend.tsx:50]: Agree, i´ll get on it 👍

**Review by bjosttveit — APPROVED** [review]
## Very nice
I only had one comment about global css use, but since this PR has dragged on for a while you can leave just it for later if you want to get it through now

**Pull request merged**